### PR TITLE
Add auth template to monorepo

### PR DESCRIPTION
**Pull request by djbarnwal**
Move old Auth template to svelte

Tagging @ericpgreen2 for web-admin/CTA components related changes and @hamilton for overall structure of the package. 

- [x] CI dependent on #2361

### CODE REVIEW

File: web-auth/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		"svelte-check": "^3.0.1",
+		"tailwindcss": "^3.2.7",
+		"tslib": "^2.4.1",
+		"typescript": "^4.9.3",
+		"vite": "^4.0.0"
+	},
+	"type": "module",
+	"dependencies": {
+		"auth0-js": "^9.20.2",
+		"dotenv": "^16.0.3"
```
Comment (ericpgreen2) [inline web-auth/package.json:46]: Can these be `devDependencies`? We should only use `devDependencies` if possible.

File: .prettierignore [hunk 2]
```diff
@@ -49,3 +49,7 @@ sql/
 
 # a temporary fix until we refactor css classes
 *.css
+
+# generated by runtime
```
Comment (ericpgreen2) [inline .prettierignore:53]: generated by Orval

File: web-auth/.gitignore
Comment (ericpgreen2) [inline web-auth/.gitignore:1]: This looks like the same `.gitignore` that's at the top-level? Can you trim this down to only ignore things relevant to `web-auth`?

File: web-auth/src/app.css
Comment (ericpgreen2) [inline web-auth/src/app.css:1]: Can you instead use the `RillTheme` component, which imports `web-common/app.css`, so we don't need this file?

File: web-auth/package.json (hunk 1 above)
Comment (djbarnwal) [reply web-auth/package.json:46]: Auth0 is a core dependency and it's APIs are directly used in the codebase. Have moved `dotenv` to `devDependencies`

**Comment by ericpgreen2** [comment]
Ah one more thing. Can we add `web-auth` to `.github/web-test.yml` our Github Action configuration? We can mirror the same format as the other `web-x` directories.

**Review by ericpgreen2 — APPROVED** [review]

**Pull request merged**